oxidation/parsec_cli/parsec_sequestre_dump/src/main.rs

```diff
@@ -0,0 +1,159 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+#[macro_use]
+extern crate diesel;
+
+mod model;
+mod schema;
+
+use clap::Parser;
+use diesel::{Connection, QueryDsl, RunQueryDsl, SqliteConnection};
+use parsec_api_crypto::SecretKey;
+
+use crate::{
+    model::{Block, Data, VlobAtom},
+    schema::{block, device, realm_role, user_ as user, vlob_atom},
+};
+
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
```

Comment:
could you add a *help* message to the options, It would be useful for the user when he make a mistake or use `--help` / `-h`